core/lib/instrument_api.h

```diff
@@ -1812,6 +1811,14 @@ DR_API
 process_id_t
 dr_get_process_id(void);
 
+DR_API
+/** Returns the process id of the process associated with drcontext \p drcontext.
```

Comment:
style: newline after `/**` (see https://github.com/DynamoRIO/dynamorio/wiki/Code-Style#commenting-conventions number 4; see also the surrounding code)

Reply:
Is there a way to check for things lik this automatically?
I run `clang-format`, but it didn't change this comment.
(I also see there are `vera++` checks, but they seem to fail on code unrelated to my changes if enabled.)

Reply:
Done.

Reply:
`vera++` is enabled on Travis and my local machine and there are no failures there -- what failure do you see (unsure how it would be different: never seen vera discrepancies before)?

Reply:
> Is there a way to check for things lik this automatically?
> I run `clang-format`, but it didn't change this comment.

Unfortunately there is no clang-format rule available for that particular style AFAIK.  A `vera++` one could perhaps be written.  The style choices were made before clang-format existed, and already some were dropped or switched for better automated enforcement, though this one is still there.

Reply:
>what failure do you see (unsure how it would be different: never seen vera discrepancies before)?

Turns out it starts checking CMake-generated files in the build directory and fails.
```
[  0%] Checking style with vera++ in build/CMakeFiles/3.10.2/CompilerIdC
/home/we/dynamorio/build/CMakeFiles/3.10.2/CompilerIdC/CMakeCCompilerId.c:217: error line_length_content: Line too long: exceeds 90 chars.
/home/we/dynamorio/build/CMakeFiles/3.10.2/CompilerIdC/CMakeCCompilerId.c:339: error line_length_content: Line too long: exceeds 90 chars.
```
This doesn't happen with out-of-source builds, but I've been building exactly like the [wiki](https://github.com/DynamoRIO/dynamorio/wiki/How-To-Build#linux) says.

Reply:
I see, so some glob somewhere looks inside the build dir if it's a child of the top-level source.